hs_core/resourcemap_urls.py

```diff
@@ -0,0 +1,14 @@
+__author__ = 'pabitra'
```

Comment:
By default PyCharm adds that when a new py file is added. Do we need to remove it?